src/openfermion/ops/__init__.py

```diff
@@ -15,9 +15,7 @@
                                 normal_ordered,
                                 number_operator)
 from ._qubit_operator import QubitOperator
-from ._polynomial_tensor import (PolynomialTensor,
-                                 one_body_basis_change,
-                                 two_body_basis_change)
+from ._polynomial_tensor import PolynomialTensor
```

Comment:
we like to have alphabetically organized imports when possible.

Reply:
Sure, but I didn't introduce this problem. If the person originally writing this code didn't have any particular reason to do it this way, I will change that.

Reply:
It turns out that I can't change the order freely, because some files imported later depends on some earlier imports having been processed.